src/legacy/core_plugins/data/public/index_patterns/index_patterns/index_pattern.test.ts

```diff
@@ -109,27 +110,14 @@ const apiClient = {
   getFieldsForWildcard: jest.fn(),
 };
 
-const notifications = {
```

Comment:
leave this defined at top level and call it notificationsServiceMock

Reply:
Great point! Thanks, fixed